app/components/common/AntagonistsComp/AntagonistsComp.jsx

```diff
@@ -5,6 +5,8 @@ require.context("./img", false, /^\.\/.*\.jpg$/); /*eslint no-undef:0*/
 
 import BigBattle from '../BigBattle/BigBattle.jsx';
 import json from 'json!./antagonists.json';
+// ["Arya Stark","Thoros of Myr"],  no plod
```

Comment:
I don't see the point why leaving not understandable comments in some js files :wink: 
If we wanna add it at some point, we just add it.